microsoft-edge/devtools-guide.md

```diff
@@ -75,7 +69,7 @@ These shortcuts control the main DevTools window and should work across all tool
 |:--- |:--- |  
 | Show/Hide DevTools \(opens to last viewed panel\) | `F12`, `Ctrl`+`Shift`+`I` |  
 | Toggle docking \(Undock/Bottom/Right\) | `Ctrl`+`Shift`+`D` |  
-| Open file | `Ctrl`+`P`, `Ctrl`+`O` |  
```

Comment:
This isn't actually correct for the latest version of the EdgeHTML-based browser. See: https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide/whats-new#source-file-search-tools. The other changes are fine but I wouldn't make this change. Instead I would let the customer know that if they update Windows to the latest version, this shortcut will work as documented.

Reply:
Updated and added note to clarify Windows validation for shortcuts.